packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -557,3 +558,34 @@ export const announceCustomStyles = ({
     document.head.appendChild(link)
   })
 }
+
+const onSSEProcessingFinishedEvent = ({
+  store,
+  msg
+}: {
+  store: Store<unknown>
+  msg: MessageEvent
+}): void => {
+  try {
+    const postProcessingData = JSON.parse(msg.data)
+    store.commit('Files/UPDATE_RESOURCE_FIELD', {
+      id: postProcessingData.itemid,
+      field: 'processing',
+      value: false
+    })
+  } catch (_) {
+    console.error('Unable to parse sse postprocessing data')
+  }
+}
+
+export const registerSSEEventListeners = ({
+  store,
+  clientService
+}: {
+  store: Store<unknown>
+  clientService: ClientService
+}): void => {
+  clientService.sseAuthenticated.addEventListener(MESSAGE_TYPE.POSTPROCESSING_FINISHED, (msg) =>
+    onSSEProcessingFinishedEvent({ store, msg })
+  )
```

Comment:
Is there a reason for the post-processing SSE registration + handling to live in the runtime? Isn't that logic bound to the files app?

Reply:
Yes we talked about this. We want to make this as global as possible. For example if you open a file in a editor (which is locked) it might be disabled to edit and will be indicated trough the app and indicates as editable as soon the locking-released event was fired.